CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### 9.0.2: February, 2021
```

Comment:
Heads-up, since the last version was 9.0.10, this should probably be either 9.0.11 or 9.1.0.

Reply:
Indeed, I have to take a look at this (and change all the references to it).